plugins/template.md

```diff
@@ -212,7 +212,7 @@ And that will be changed to:
 
 ### `template_preview_replace_values`
 
-This is an object containing items that will be replaced with their respective string values in the template preview shown in the template dialog.
+This is an object containing items that will be replaced with their respective string values in the template preview shown in the template dialog, **but will not replaced when a template is inserted into the editor content**.
```

Comment:
For what it's worth, this functionality confused me when I initially implemented it, hence my emphasis